build.gradle

```diff
@@ -88,7 +88,7 @@ allprojects {
     }*/
     maven { url 'https://mapbox.bintray.com/mapbox' }
     // uncomment if snapshots access is needed
-    // maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
+     maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
```

Comment:
be committed again, when mapbox-java is updated

Reply:
Could we avoid merging snapshot repo references to the main branch? It's bitten us in the past as it slipped into a release. I'd rather have a snapshot tied to a feature PR that needs it and then bump the version of mapbox-java before merging. Alternatively, we can release a `-beta` of mapbox-java.

Reply:
@LukasPaczos we aren't gonna merge PR with current changes, this change is here to show how it work and unblock @langsmith UI side work. We definitely gonna make a new mapbox-java release if everything is 🆗